sonar-project.properties

```diff
@@ -1,6 +1,6 @@
 sonar.projectKey = yoti-web-sdk:ruby
 sonar.projectName = ruby-sdk
-sonar.projectVersion = 1.6.1
+sonar.projectVersion = 1.6.2
```

Comment:
Not relevant to this PR but I think going forward we plan to change development branch's version-strings to the next/proposed version in the PR that updates development after a release.

Reply:
Fixed in f26aefe